demo/client/src/3d-demo.ts

```diff
@@ -472,6 +483,15 @@ class ThreeDRenderer extends Renderer {
         this.renderContext.gl.uniform4f(uniforms.uHints, 0, 0, 0, 0);
     }
 
+    redrawVR(frame: VRFrameData): void {
+        this.vrProjectionMatrix = frame.leftProjectionMatrix;
+        this.camera.setView(F32ArrayToMat4(frame.leftViewMatrix), frame.pose);
+        this.redraw();
+        this.vrProjectionMatrix = frame.rightProjectionMatrix;
+        this.camera.setView(F32ArrayToMat4(frame.rightViewMatrix), frame.pose);
+        this.redraw();
```

Comment:
Could you add a comment here specifying why you need to redraw twice? Also I believe you can replace `F32ArrayToMat4` with `glmatrix.mat4.clone()`, as above.